libraries/Microsoft.Bot.Connector/ConversationsEx.cs

```diff
@@ -40,10 +40,6 @@ public static async Task<ConversationResourceResponse> CreateDirectConversationA
         {
             var result = await operations.CreateConversationWithHttpMessagesAsync(GetDirectParameters(bot, user, activity), null, cancellationToken).ConfigureAwait(false);
             var res = result.Body;
```

Comment:
>var res = result.Body; [](start = 12, length = 22)

Should we remove this assignment and just return result.Body?

Reply:
that would be cleaner - sorry should have seen that - 2 mins